src/components/dds/Notification/Notification.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import getRootClassNames from '../common/getRootClassNames';
+import { componentClasses } from 'dds-core/components/notification/notification';
+import { Button } from 'components';
+import 'dds-core/components/notification/_notification.scss';
+import { Icon } from '../Icon';
+import { useState } from 'react';
+
+export interface NotificationProps extends React.HTMLAttributes<HTMLDivElement> {
+    /**
+     * Title text for the notification
+     * */
+    title: string;
+    /**
+     * Message body for the notification
+     * */
+    messageBody: string;
+    /**
+     * Time stamp for the notification
+     * */
+    timeStamp: string;
```

Comment:
I think it is commonly used as a single world:
```suggestion
    /**
     * Timestamp for the notification
     * */
    timestamp: string;
```

Reply:
changed: https://github.com/cloudify-cosmo/cloudify-ui-components/pull/228/commits/67fc8954618e0dafc3fbde776ab900ae3af2c3df